pkg/systemd/services/services.jsx

```diff
@@ -65,6 +64,11 @@ export let clock_monotonic_now;
 
 export const MAX_UINT64 = 2 ** 64 - 1;
 
+function debug() {
+    if (window.debugging == "all" || window.debugging?.includes("services"))
+        console.debug.apply(console, arguments);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230308-151253-318e440b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)